packages/filesystem/src/browser/filesystem-frontend-contribution.ts

```diff
@@ -16,29 +16,19 @@
 
 import { injectable, inject } from 'inversify';
 import URI from '@theia/core/lib/common/uri';
-import { environment } from '@theia/application-package/lib/environment';
-import { MaybePromise, SelectionService, isCancelled } from '@theia/core/lib/common';
-import { Command, CommandContribution, CommandRegistry } from '@theia/core/lib/common/command';
+import { MaybePromise } from '@theia/core/lib/common';
+import { CommandContribution, CommandRegistry } from '@theia/core/lib/common/command';
 import {
     FrontendApplicationContribution, ApplicationShell,
     NavigatableWidget, NavigatableWidgetOptions,
-    Saveable, WidgetManager, StatefulWidget, FrontendApplication, ExpandableTreeNode
+    Saveable, WidgetManager, StatefulWidget, FrontendApplication
 } from '@theia/core/lib/browser';
 import { FileSystemWatcher, FileChangeEvent, FileMoveEvent, FileChangeType } from './filesystem-watcher';
 import { MimeService } from '@theia/core/lib/browser/mime-service';
-import { TreeWidgetSelection } from '@theia/core/lib/browser/tree/tree-widget-selection';
 import { FileSystemPreferences } from './filesystem-preferences';
-import { FileSelection } from './file-selection';
-import { FileUploadService } from './file-upload-service';
 
 export namespace FileSystemCommands {
 
-    export const UPLOAD: Command = {
```

Comment:
This is a breaking change to anyone referring to `FileSystemCommands.UPLOAD`.
It should be noted in our `CHANGELOG` under breaking changes.